LICENSE

```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
This change is invalid as it should be "[""]" according to https://www.apache.org/licenses/LICENSE-2.0